tests/testthat/test-pmx_nlmixr.R

```diff
@@ -46,21 +46,21 @@ if (helper_skip() && requireNamespace("nlmixr2", quietly = TRUE)) {
     p14 <- ctr %>% pmx_plot_eta_hist()
     p15 <- ctr %>% pmx_plot_eta_matrix()
 
-    expect_true(inherits(p1, c("gg", "ggplot")))
-    expect_true(inherits(p2, c("gg", "ggplot")))
-    expect_true(inherits(p3, c("gg", "ggplot")))
-    expect_true(inherits(p4, c("gg", "ggplot")))
-    expect_true(inherits(p5, c("gg", "ggplot")))
-    expect_true(inherits(p6, c("gg", "ggplot")))
-    expect_true(inherits(p7, c("gg", "ggplot")))
-    expect_true(inherits(p8, c("gg", "ggplot")))
-    expect_true(inherits(p9, c("gg", "ggplot")))
-    expect_true(inherits(p10, c("gg", "ggplot")))
+    expect_true(inherits(p1, c("ggplot", "ggplot2::ggplot")))
```

Comment:
Are these tests currently disabled? If not, do they work on older ggplot2 versions? Wondering if they need to be disabled until the ggplot2 update is released